translations/de.ts

```diff
@@ -1037,7 +1041,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/widget/form/settings/generalsettings.ui" line="119"/>
         <source>qTox will start minimized in tray.</source>
         <comment>toolTip for Start in tray setting</comment>
-        <translation>qTox minimiert starten.</translation>
+        <translation>qTox wird minimiert starten.</translation>
```

Comment:
Jep, genau. Als Entwickler mit Blick für die User (**hust** :smile:) achte ich darauf, möglichst einfache Sätze (oder Stichworte) im "Präsenz" zu verwenden. In diesem Fall etwa so: "Wenn ich diesen Knopf drücke, bedeutet das, dass **qTox (in die Taskleiste) minimiert startet**."

Problem ist: Bei meinem (und vielen anderen: OSX, Elementary OS, ... ) System gibt es gar keine Taskleiste. Deshalb müsste dort nicht nur Sprachlich, sondern auch Umgebungstechnisch übersetzt werden - schwierige Kiste. Ich find's am einfachsten (und deutlichsten), wenn die Beschriftung hier einfach sagt: "qTox startet minimiert". Das heißt, wenn der Knopf **nicht** gedrückt ist, startet qTox auch **nicht** minimiert. Was mir gerade noch einfällt ... was hältst Du von "qTox beim start ausblenden/minimieren". Gibt's hier eigentlich einen Tooltip, der das genauer beschreibt? Dort könnte man dann auch erwähnen, dass das abhängig vom verwendeten OS bzw. DE ist.

Beispiel (für englischen orig. text):
"When enabled, qTox will be started invisible. Depending on your operating system's desktop environment it appears minimized to the tray/dock."

Sorry für die lange Antwort, aber ich denke du hast eine Idee, was ich sagen will. Übersetzung ist halt auch ein bissle Überzeugungsarbeit. Danke für deine tolle Hilfe. :+1: :)